packages/react-dom-bindings/src/client/ReactDOMComponent.js

```diff
@@ -1018,6 +794,17 @@ export function setInitialProperties(
   // TODO: Make sure that we check isMounted before firing any of these events.
 
   switch (tag) {
+    case 'div':
+    case 'span':
+    case 'svg':
+    case 'path':
+    case 'a':
+    case 'g':
+    case 'p':
+    case 'li': {
```

Comment:
https://almanac.httparchive.org/en/2021/markup#top-elements
looks like `i` and `ul` are also popular… wonder if it's worth including them? and moving the img/link special case up? and maybe adding meta? (does float go through this path?)

Reply:
I guess they're less popular. Didn't show up in my testing. It's interesting that groupings like `ul` and `tbody` or even `tr` inherently are less common than the n items below them.

Float doesn't go through this path.

I tried to keep things somewhat grouped together in groups that make sense. Could maybe group all void elements but I think we're going to end up special casing `img` soon for suspensey images so then we can move it up.